config/config.go

```diff
@@ -124,6 +128,12 @@ func mergeConfigFromFile(configpath string, source Local) (Local, error) {
 		source.Archival = true
 		source.EnableLedgerService = true
 		source.EnableBlockService = true
+
+		// If gossip fanout has not been explicitly overridden, use defaultRelayGossipFanout
+		// rather then the default gossip fanout setting from defaultLocal
+		if source.GossipFanout == defaultLocal.GossipFanout {
```

Comment:
does this prevent relays from explicitly overriding this fanout change back the current local default value of 4?

Reply:
Yes, though they could set 3 or 5.